pkg/config/model/viper.go

```diff
@@ -26,17 +26,18 @@ type Source string
 
 // Declare every known Source
 const (
-	SourceDefault      Source = "default"
-	SourceUnknown      Source = "unknown"
-	SourceFile         Source = "file"
-	SourceEnvVar       Source = "environment-variable"
-	SourceAgentRuntime Source = "agent-runtime"
-	SourceRC           Source = "remote-config"
-	SourceCLI          Source = "cli"
+	SourceDefault            Source = "default"
+	SourceUnknown            Source = "unknown"
+	SourceFile               Source = "file"
+	SourceEnvVar             Source = "environment-variable"
+	SourceLocalConfigProcess Source = "local-config-process"
+	SourceAgentRuntime       Source = "agent-runtime"
+	SourceRC                 Source = "remote-config"
+	SourceCLI                Source = "cli"
 )
 
 // sources list the known sources, following the order of hierarchy between them
-var sources = []Source{SourceDefault, SourceUnknown, SourceFile, SourceEnvVar, SourceAgentRuntime, SourceRC, SourceCLI}
+var sources = []Source{SourceDefault, SourceUnknown, SourceFile, SourceEnvVar, SourceLocalConfigProcess, SourceAgentRuntime, SourceRC, SourceCLI}
```

Comment:
We need to update the comp/metadata/inventoryagent payload to reflect the new source.

Reply:
Done in [`311b9ee` (#23211)](https://github.com/DataDog/datadog-agent/pull/23211/commits/311b9eed539b67425be1989f0c0a25117dddfeb7)